guide/interactions/buttons.md

```diff
@@ -0,0 +1,203 @@
+# Buttons
+
+With the components API, you can create interactive message components. In this page, we'll be focusing on how to send, receive and respond to buttons using discord.js!
+
+
+## Building and sending buttons
+
+Buttons are part of the `MessageComponent` class, which can be sent via messages or interaction responses. A button, as any other message component, must be in an `ActionRow`.
+
+::: warning
+You can have a maximum of:
+- five `ActionRows` per message
+- five buttons within an `ActionRow`
+:::
+
+Now, to create a button we use the `MessageActionRow()` and `MessageButton()` builder functions and then pass the resulting object to `CommandInteraction#reply()` as `InteractionReplyOptions` like this:
+
+```js {1,7-11,13}
+const { MessageActionRow, MessageButton } = require('discord.js');
+
+client.on('interaction', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'ping') {
+		const row = new MessageActionRow()
+			.addComponent(new MessageButton()
```

Comment:
```suggestion
			.addComponents(new MessageButton()
```
I removed `addComponent` because it didn't do anything differently except limit to 1 entry, it now only has `addComponents` which can just take 1 normally